templates/web/base/admin/reports/edit.html

```diff
@@ -181,21 +167,49 @@
                 <span>zoom</span>
             </a>
         </div>
-        <input type="submit" name="rotate_photo_[% loop.index %]" value="[% loc('Rotate Left') %]">
-        <input type="submit" name="rotate_photo_[% loop.index %]" value="[% loc('Rotate Right') %]">
+        <input class="btn" type="submit" name="rotate_photo_[% loop.index %]" value="[% loc('Rotate Left') %]">
+        <input class="btn" type="submit" name="rotate_photo_[% loop.index %]" value="[% loc('Rotate Right') %]">
         <br><input type="checkbox" id="remove_photo_[% loop.index %]" name="remove_photo_[% loop.index %]" value="1">
         <label class="inline" for="remove_photo_[% loop.index %]">[% loc("Remove photo (can't be undone!)") %]</label></li>
     </li>
   [% END %]
 </ul>
 </li>
 [% END %]
+</ul>
 
-<li><label class="inline-text" for="closed_updates">[% loc('Closed to updates') %]:</label>
-    <input type="checkbox" id="closed_updates" name="closed_updates"[% ' checked' IF problem.extra.closed_updates %]></li>
+<div class="clearfix full-width" style="background-color: #eee; padding: 1em 1em 0;">
```

Comment:
FWIW I’m totally fine with these inline CSS styles. Barely seems worth breaking them out into a separate class, given this is just an admin page, and the whole form is already quite rough-and-ready, it feels like we can get away with it here.